src/javascript/app/pages/trade/tick_trade.js

```diff
@@ -244,7 +243,7 @@ const TickDisplay = (() => {
             // round calculated barrier
             const calc_barrier = (total / applicable_ticks.length).toFixed(parseInt(display_decimals) + 1);
             drawBarrier({ value: calc_barrier, text: 'Average' });
-            contract_barrier_label = getContractBarrierLabel('Average', calc_barrier);
+            contract_barrier_label = getContractBarrierLabel('Barrier', calc_barrier);
```

Comment:
we're changing the label to barrier?

Reply:
for Asians trade types,  it is currently labelled as Barrier in the overlay. but i can change it to Average instead

Reply:
let's keep it :)